processes.py

```diff
@@ -129,7 +131,7 @@ def process_to_dict(process):
       'Further standardization.',
       ['Intent to Ship email',
        'Request to migrate incubation',
-       'TAG issues addressed',
+       'TAG review completed',
```

Comment:
The important part here is that the issues have been addressed, not that the review itself has been completed.  I'd revert this.

Reply:
Done.